Dockerfile

```diff
@@ -12,11 +12,17 @@ RUN mkdir -p ${GOPATH}/src ${GOPATH}/bin
 # Install Go Tools
 RUN GO111MODULE= go get -u golang.org/x/lint/golint
 
+# Install Linkerd
+RUN curl -Lo /usr/local/bin/linkerd https://github.com/linkerd/linkerd2/releases/download/stable-2.1.0/linkerd2-cli-stable-2.1.0-linux
```

Comment:
can only recommend to verify the checksum

Reply:
I thought about this, but what would us protect this from?

Reply:
broken downloads :)

Reply:
fixed